include/cantera/zeroD/ReactorBase.h

```diff
@@ -84,6 +94,9 @@ class ReactorBase
     //!              ReactorBase with Solution object.
     void setSolution(shared_ptr<Solution> sol);
 
+    //! Access the Solution object used to represent the contents of this reactor.
+    shared_ptr<Solution> solution() { return m_solution; }
```

Comment:
In other contexts (specifically Python), we use `contents`. I don't have strong opinions, though.

Reply:
Unfortunately, `ReactorBase::contents` already exists, returning a `ThermoPhase&`.

I think getting to a consistent state with `contents` as the name will take a couple of iterations (versions) -- we can deprecate the existing `ReactorBase::contents` now, along with the Python `ReactorBase.thermo` and `ReactorSurface.kinetics` properties, but at least for now we need a distinct name for this method.